### PR TITLE
Add support to select the attendee list and enable media capture for the selected attendees.

### PR DESCRIPTION
**Pull request by akshay-prabhu-github**
**Issue #:**
None.

**Description of changes:**
Added support for selecting the attendees from the roster and enabling media capture for the selected attendees. This feature is useful, while testing media capture for certain selected number of attendees.

**Testing:**
Below is a snapshot of the attendee for whom we intend to capture the media.
Screen Shot 2022-06-02 at 11 04 37 PM

**Can these tested using a demo application?**
Yes

**Please provide reproducible step-by-step instructions.**
1. Launch the demo application in browser.
2. Select the attendee that you want to capture the media for
3. Click the Media capture option to start recording the media for that particular attendee.
4. Make sure the Media Pipeline attendee joins the meeting.

**Checklist:**

1. Have you successfully run `npm run build:release` locally?
Yes

5. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved?
No

6. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved?
No

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: demos/browser/app/meetingV2/meetingV2.ts [hunk 1]
```diff
@@ -236,6 +236,8 @@ export class DemoMeetingApp
   static readonly MAX_MEETING_HISTORY_MS: number = 5 * 60 * 1000;
   static readonly DATA_MESSAGE_TOPIC: string = 'chat';
   static readonly DATA_MESSAGE_LIFETIME_MS: number = 300_000;
+  
+  static readonly MAX_ATTENDEES_TO_SELECT: number = 25;
```
Comment (ltrung) [inline demos/browser/app/meetingV2/meetingV2.ts:240]: Can you comment on why only 25?

File: demos/browser/app/meetingV2/meetingV2.ts [hunk 2]
```diff
@@ -1927,16 +1934,27 @@ export class DemoMeetingApp
...
     }
     while (roster.getElementsByTagName('li').length > newRosterCount) {
       roster.removeChild(roster.getElementsByTagName('li')[0]);
     }
+
+    // Delete the selected attendees that are no longer part  of the roster
+    for (let attendeeId of this.selectedAttendees) {
+        if (this.roster[attendeeId] == undefined) {
+          this.selectedAttendees.delete(attendeeId);
+          this.meetingLogger.error(attendeeId);
```
Comment (ltrung) [inline demos/browser/app/meetingV2/meetingV2.ts:1948]: Why do we log error here?

File: demos/browser/server.js [hunk 3]
```diff
@@ -192,9 +192,43 @@ function serve(host = '127.0.0.1:8080') {
...
+            Content: {
+              State: "Enabled",
+              MuxType: "ContentOnly"
+            }
+          }
+        }
+      
+        if (attendeeIds.length > 0) {
+          chimeSDKConfig["SourceConfiguration"] = {
+            SelectedVideoStreams : {
```
Comment (ltrung) [inline demos/browser/server.js:222]: Is it a new feature recently released?

File: demos/browser/app/meetingV2/meetingV2.ts (hunk 1 above)
Comment (akshay-prabhu-github) [reply demos/browser/app/meetingV2/meetingV2.ts:240]: will do

File: demos/browser/app/meetingV2/meetingV2.ts (hunk 2 above)
Comment (akshay-prabhu-github) [reply demos/browser/app/meetingV2/meetingV2.ts:1948]: Good catch! left that accidentally there while debugging. Will remove it.

File: demos/browser/server.js (hunk 3 above)
Comment (akshay-prabhu-github) [reply demos/browser/server.js:222]: This feature was launched with Media Capture pipelines, but the support for selection of attendees was not added to the meeting demo. This code review intends to add that support to the Meeting Demo.

Reference to the CreateMediaCapturePipeline: https://docs.aws.amazon.com/chime/latest/APIReference/API_CreateMediaCapturePipeline.html

**Pull request closed without merging**